src/lib.rs

```diff
@@ -337,8 +336,12 @@ fn page_completions<C: Candidate, H: Helper>(
             if i < candidates.len() {
                 let candidate = &candidates[i].display();
                 let width = candidate.width();
-                if let Some(highlighter) = s.highlighter() {
-                    ab.push_str(&highlighter.highlight_candidate(candidate, CompletionType::List));
+                if s.out.colors_enabled() {
```

Comment:
Wrong: duplicated code (previously this logic was factorized in `s.highlighter()` method).